backend/src/scripts/whitelisting/get_all_templates.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config()
+require('module-alias/register')
+
+import sequelizeLoader from '@core/loaders/sequelize.loader'
+import { GovsgTemplate } from '@govsg/models'
+import { Parser } from '@json2csv/plainjs'
+import { string as stringFormatter } from '@json2csv/formatters'
+import * as fs from 'fs'
+
+/**
+ * Returns a csv file of all templates in the database.
+ * The fields of the csv file returned are as follows:
+ * templateId, templateLabel, templateName, multilingualSupport
+ */
+void (async function main() {
+  try {
+    await sequelizeLoader()
+    const templates = await GovsgTemplate.findAll()
+
+    const opts = {
+      fields: [
+        'templateId',
+        'templateLabel',
+        'templateName',
+        'multilingualSupport',
```

Comment:
Have you tested this script with example values of `multilingualSupport`? Its type is `GovsgTemplateLanguageMetadata`.

Reply:
I've not tested it against any example values yet. I tried to reference prod and staging db but the column has no fields. 

I wrongly assumed that the field's type was `string[]`. I think it would make sense for me to return an array of `GovsgTemplateLanguageMetadata.languageCode` in the csv?

Reply:
Shucks, no examples around.

I suppose `multilingualSupport` is something to be updated by ourselves since template information is still hardcoded into db.

Yep, either `language*` works for me. But it's also more critical to have `languageCode` populated in the db.

Let's go with your suggestion.